src/plugins/dashboard/public/dashboard_strings.ts

```diff
@@ -318,3 +340,23 @@ export const dashboardListingTable = {
       defaultMessage: 'Description',
     }),
 };
+
+export const dashboardUnsavedListingStrings = {
+  getUnsavedChangesTitle: (plural = false) =>
+    i18n.translate('dashboard.listing.unsaved.unsavedChangesTitle', {
+      defaultMessage: 'You have unsaved changes in the following {dash}',
+      values: {
+        dash: plural
+          ? dashboardListingTable.getEntityNamePlural()
+          : dashboardListingTable.getEntityName(),
+      },
+    }),
+  getEditTitle: () =>
+    i18n.translate('dashboard.listing.unsaved.editTitle', {
+      defaultMessage: 'Continue Editing',
```

Comment:
nit but I think we'll want to use sentence case rather than title case

Reply:
Whoops! Good call